src/tests/JIT/Regression/JitBlue/GitHub_68136/GitHub_68136.cs

```diff
@@ -0,0 +1,34 @@
+public class Program
```

Comment:
This needs a license header.

Reply:
Also, you should rename the file and class name here to `Runtime_68136` (the GitHub_XYZ issues refer to issues when the repo was at dotnet/coreclr).